pkg/agent/controller/endpoint.go

```diff
@@ -87,7 +109,7 @@ func (e *EndpointController) endpointSliceFromEndpoints(endpoints *corev1.Endpoi
 		lhconstants.LabelSourceName:        e.serviceName,
 	}
 	endpointSlice.OwnerReferences = []metav1.OwnerReference{{
-		APIVersion:         "lighthouse.submariner.io.v2alpha1",
+		APIVersion:         "multicluster.x-k8s.io.v1alpha1",
```

Comment:
Do we need the `OwnerReference` since we have to manually delete anyway?

Reply:
> Do we need the `OwnerReference` since we have to manually delete anyway?

Useful when we later get ServiceImport aggregation.

Reply:
Also, we set the controller flag to false here, so that the endpointslice controller will not  modify/delete this object